packages/gatsby-theme-styleguide/src/gatsby-components/TableOfContents/TableOfContents.js

```diff
@@ -30,10 +30,11 @@ export default class TableOfContents extends PureComponent {
   }
 
   getAnchorPosition = e => {
-    const scrollValue =
-      document.getElementById('scroller').scrollTop +
-      document.getElementById('page_tabs_menu').clientHeight +
-      20
+    const scrollValue = document.getElementById('page_tabs_menu')
+      ? document.getElementById('scroller').scrollTop +
+        document.getElementById('page_tabs_menu').clientHeight +
+        20
+      : 20
 
     const anchors = document.querySelectorAll('.anchor-toc')
```

Comment:
I think that the corrections made in this file are not directly related to RA. Therefore, it might be better to make this file go to another dedicated PR, right?